examples/tests/sdk_tests/website/react-website/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "react-website",
```

Comment:
we need a folder with package.json in order to mock react for the tests (instead of handling with a heavy real react project in our repo- that isn't really needed)